modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/expcatalog/ProcessErrorEntity.java

```diff
@@ -21,24 +21,52 @@
 package org.apache.airavata.registry.core.entities.expcatalog;
 
 import javax.persistence.*;
+import java.io.Serializable;
+import java.sql.Timestamp;
 import java.util.List;
 
+/**
+ * The persistent class for the process_error database table.
+ */
 @Entity
-@Table(name = "EXPCAT_PROCESS_ERROR")
+@Table(name = "PROCESS_ERROR")
 @IdClass(ProcessErrorPK.class)
-public class ProcessErrorEntity {
+public class ProcessErrorEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name = "ERROR_ID")
     private String errorId;
+
+    @Id
+    @Column(name = "PROCESS_ID")
     private String processId;
-    private long creationTime;
+
+    @Column(name = "CREATION_TIME")
+    private Timestamp creationTime;
+
+    @Lob
+    @Column(name = "ACTUAL_ERROR_MESSAGE")
     private String actualErrorMessage;
+
+    @Lob
+    @Column(name = "USER_FRIENDLY_MESSAGE")
     private String userFriendlyMessage;
+
+    @Column(name = "TRANSIENT_OR_PERSISTENT")
     private boolean transientOrPersistent;
+
+    @Lob
+    @Column(name = "ROOT_CAUSE_ERROR_ID_LIST")
     private List<String> rootCauseErrorIdList;
```

Comment:
I think this is a comma separated list. It would just map to a String not a List<String>.

Reply:
Is this the same for ExperimentError and TaskError?